.pre-commit-config.yaml

```diff
@@ -33,9 +33,10 @@ repos:
                "--ignore-path", "docs/source/api/"]
 
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: "v0.812"
+    rev: v0.910
     hooks:
       - id: mypy
+        exclude: "asv_bench"
         args:
           - --ignore-missing-imports
           - --follow-imports=silent
```

Comment:
Think we can add `- --install-types` here

Reply:
Added

Reply:
Fails https://results.pre-commit.ci/run/github/137399755/1641332894.ZbJaSIU_RT6v8vHTUWDumQ

Reply:
Do we need install-types?